src/parser/jobs/pld/modules/OGCDDowntime.tsx

```diff
@@ -1,9 +1,28 @@
 import ACTIONS from 'data/ACTIONS'
 import CooldownDowntime from 'parser/core/modules/CooldownDowntime'
 
+const ALLOWEDDOWNTIMEFOF = 4000
+const ALLOWEDDOWNTIMEREQ = 4000
+
+const FIRSTUSEOFFSETFOF = 24000
```

Comment:
Probably worth sticking a comment here describing why we allow first use offset to be so big.

Reply:
Should be resolved in current version --- I also added brief explanations for the GNB numbers.